app/views/casa_cases/show.html.erb

```diff
@@ -1,4 +1,6 @@
 <div class="row">
+  <%= render "thank_you_modal" %>
```

Comment:
Shouldn't this be included only when the URL contains the query param? otherwise it will render unnecessary HTML